sklearn/utils/estimator_checks.py

```diff
@@ -1443,41 +1443,90 @@ def check_supervised_y_2d(name, estimator_orig):
     assert_allclose(y_pred.ravel(), y_pred_2d.ravel())
 
 
-@ignore_warnings(category=(DeprecationWarning, FutureWarning))
+@ignore_warnings
+def check_classifiers_predictions(X, y, name, classifier_orig):
+    classes = np.unique(y)
+    classifier = clone(classifier_orig)
+    if name == 'BernoulliNB':
+        X = X > X.mean()
+    set_random_state(classifier)
+    # fit
```

Comment:
I don't think this comment helps the reader

Reply:
I don't think either. I reused the existing code snippet, I'll fix it :)